src/_live-rendering.ts

```
@@ -419,4 +420,4 @@ if (import.meta.vitest != null) {
 				.toThrow('This operation was aborted');
 		});
 	});
-}
+}
```

ESLint

[error] Newline required at end of file but not found.

(style/eol-last)